pyfolio/tears.py

```diff
@@ -497,10 +485,11 @@ def create_returns_tear_sheet(returns, positions=None,
         If True, returns the figure that was plotted on.
     """
 
-    if benchmark_rets is None:
-        benchmark_rets = utils.get_symbol_rets('SPY')
-
-    returns = returns[returns.index > benchmark_rets.index[0]]
+    # If benchmark has been passed, and the strategy's history is longer than
```

Comment:
It looks like this logic is duplicated in a few places. We might want to consider factoring this out into a helper function named something like `clip_returns_to_benchmark`.

Reply:
Added `utils.clip_returns_to_benchmark`. This will require a unit test.